docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+Continuous Patching Workflow in Azure Container Registry
+========================================================
+
+## Introduction
+
+Continuous Patching is a feature in Azure Container Registry that allows you to recurringly scan and patch specified artifacts for only OS-level  vulnerabilities within the registry. The workflow enables to set a cadence for run and create a recurring ACR task that scans your configured list of images for vulnerabilities (CVEs) using [Trivy](https://trivy.dev/) and patch them using [Copa](https://project-copacetic.github.io/copacetic/website/).
+ 
+
+> **NOTE:**
+> Continuous Patching is a limited pilot program as of October 2024.
+
+
```

Comment:
It would also be good to note that Continuous Patching is not supported in free tier ACRs that do not have Tasks enabled.